### PR TITLE
Headline structure improvement

### LINKED ISSUE #3673: [BUG] Page "extensions": headline structure
**Describe the bug**
Page "extensions" has some issues with the headlines structure

grafik

**Expected behavior**

1. `<h1>` = Fresh RSS (check, not an issue)
2. `<h1>` should be `<h2>` = "Extensions"
3. both `<h2>` should be `<legend>` as it is the default headline of all other forms f.e. 
grafik
4. `<h2>` should be `<h3>` because it is not in a `<form>` and is a level under  `<h2>` = "Extensions" (see 2.)

**Environment information (please complete the following information):**
 - FreshRSS version: 1.18.1

**Additional context**
A very good headline structure helps styling the pages and reading the pages with screen readers

### PR DESCRIPTION
**Pull request by math-GH**
Closes #3673

Changes proposed in this pull request:
all templates:
- `<h1>` around the FreshRSS logo: removed (because it is not the main headline)
- each page has a `<h1>` as a real headline
- lot of fake `<legend>` changed to mostly `<h2>` headlines
- now we have a clear headline structure on each page

How to test the feature manually:
see on each page the headlines

Pull request checklist:
- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

**Comment by Alkarex** [comment]
Overall good, but we need to go through the themes to make sure they do not break. For instance a quick test with *Pafat*, see the impact on the logo:

Before:
image

After:
image

**Comment by math-GH** [comment]
> 
> 
> Overall good, but we need to go through the themes to make sure they do not break. For instance a quick test with _Pafat_, see the impact on the logo:

Will do it.

In Theme "Mapco" it leads us to a question regarding the "correct" FreshRSS logo.

before (original, with upper chases):
grafik

after (written in the original way "FreshRSS" and smaller font)
grafik

Question: is it ok for you to change the font next to the circle or keep the original font? Or would it make sense to update the original logo? IMHO: I like the upper case font

**Comment by math-GH** [comment]
Theme "Dark"

Before:
grafik

After:
grafik

**Comment by math-GH** [comment]
Theme "BlueLagoon"

before
grafik

after
grafik

**Comment by math-GH** [comment]
Thema "Ansum"

before
grafik

after:
grafik

**Comment by math-GH** [comment]
Theme: Alternative Dark

before:
grafik

after:
grafik

**Comment by math-GH** [comment]
Theme: Pafat

before
grafik

after
grafik

**Comment by math-GH** [comment]
Theme: Screwdriver

before:
grafik

after:
grafik

**Comment by math-GH** [comment]
Theme "Swage"

before
grafik


after
grafik

**Comment by math-GH** [comment]
@Alkarex: The logos are fixed now. See the screen snippets (made from Firefox)

**Comment by Alkarex** [comment]
Excellent job @mathContao 👍🏻 
There seems to be a remaining logo bug with *Origin-compact*
image

**Comment by math-GH** [comment]
Fixed theme "Origine-Compact" too

File: app/layout/header.phtml [hunk 1]
```diff
@@ -14,12 +14,9 @@ if (FreshRSS_Auth::accessNeedsAction()) {
...
 	<div class="item title">
-		<h1>
-			<a href="<?= _url('index', 'index') ?>">
-				<img class="logo" src="<?= _i('icon', true) ?>" alt="" />
-				<?= FreshRSS_Context::$system_conf->title ?>
-			</a>
-		</h1>
+		<a href="<?= _url('index', 'index') ?>">
+			<img class="logo" src="<?= _i('FreshRSS-logo', true) ?>" alt="" />
+		</a>
```
Comment (marienfressinaud) [inline app/layout/header.phtml:19]: I'm (finally) doing the update of my instance, and I realize this change breaks the possibility to change the name of the instance. I don't remember seeing someone complaining about that and I can manage this case differently, but I wonder why it was needed?

Also, if we keep this solution, the `alt` attribute should be set to "FreshRSS" to give a label to the screen readers.

Comment (Frenzie) [reply app/layout/header.phtml:19]: I simply overlooked it. But yes, setting aside the h1 thing I have to say I'm confused about the additional change.

Comment (Alkarex) [reply app/layout/header.phtml:19]: Suggestion of fix, a bit more powerful https://github.com/FreshRSS/FreshRSS/pull/4369

**Pull request merged**